src/main/kotlin/tech/harmonysoft/oss/gradle/release/paperwork/GradleReleasePaperworkPluginExtension.kt

```diff
@@ -15,4 +15,6 @@ interface GradleReleasePaperworkPluginExtension {
     val changeDescription: Property<(String) -> String?>
 
     val maxChangesPerRelease: Property<Int>
+
+    val tagPrefix: Property<String>
```

Comment:
what do you think about defining not a prefix, but complete pattern? E.g. current pattern is `release-%s` and a user might be able to define it as `v%s`. This way user can control all tag name aspects